rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -538,6 +545,57 @@ const FlowEditor: React.FC<{
     clearNodesStatusAndOutput();
   }, [clearNodesStatusAndOutput]);
 
+  // This is to check for the Input and Output block, if they are used they are passed to Runner ui's input and output screen
+  const getBlockInputsAndOutputs = useCallback(() => {
+    const inputBlocks = nodes.filter(
+      (node) => node.data.block_id === "c0a8e994-ebf1-4a9c-a4d8-89d09c86741b",
+    );
+
+    const outputBlocks = nodes.filter(
+      (node) => node.data.block_id === "363ae599-353e-4804-937e-b2ee3cef3da4",
+    );
```

Comment:
I feel there is a better way to go about this but i wasnt sure if there was, if any one has any advice/ideas please let me know, this is just to check if there are input/output blocks in the UI

Reply:
I'd just throw this in a util function. Filter seems fine, but can you filter on something that's not a string? is there a more generic way? I'd think this would be useful elsewhere if its built like reduce but targeted towards blocks maybe?

```ts
export function filterBlocksByType(blockList: Block[], select: Block){
	return 
}
```

Reply:
I did all the changes mentioned, thank you for the review!